src/news/2021-05-08-Inverted-Wordles.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Inverted Wordles
+filename: Inverted-Wordles
+date: '2021-05-08'
+---
+
+In a traditional word cloud, the most popular answers (e.g. words, phrases) are emphasized. The result of this emphasis
+is for the minority to fade away into the background. With their work on Inverted Wordles, the We Count team is seaking
+a way to invert that relationship and provide a means to amplify those minority voices.
+
```

Comment:
A copy of the Inverted Wordle image would be useful here https://wecount.inclusivedesign.ca/uploads/wordle.png?__WB_REVISION__=4d8d8b940e79ffb2a0ead11810795383

Make sure to add alt text, and a `<caption>` like in this other Floe news post http://localhost:8080/news/2020-07-02-WeCountUpdate.html